### DISCUSSION TITLE
Proposal: size-label-action

### DISCUSSION THREAD

**Discussion by splincode** [taiga-family/taiga-ui#9326] [General]
@taiga-family/core-team 

https://github.com/pascalgn/size-label-action

<img width="534" alt="image" src="https://github.com/user-attachments/assets/7ca13d50-c1a0-4e31-9726-5126d7eb6a37">
<img width="614" alt="image" src="https://github.com/user-attachments/assets/966d69f4-3be6-4446-b3d6-d26806b5bef2">

is this action necessary? would it be useful in a review?

**Comment by waterplea** [comment]
I'm not sure we want to enforce that
